test/rekt/kafka_source_test.go

```diff
@@ -59,7 +60,9 @@ func TestKafkaSource(t *testing.T) {
 		t.Fatalf("could not copy secret(%s): %v", kafkaSASLSecret, err)
 	}
 
-	for _, f := range kafkasource.DataPlaneDelivery() {
+	testingFeatures := kafkasource.DataPlaneDelivery()
+	testingFeatures = append(testingFeatures, kafkasource.MtDataPlaneDelivery(kafkaClient))
```

Comment:
This feature is only applicable when the KafkaSource under test is the multi-tenant one. Check if `os.Getenv("TEST_MT_SOURCE")` is set and add the MT features.